integration/hive/src/main/scala/org/apache/carbondata/hiveexample/HiveExample.scala

```diff
@@ -85,25 +82,35 @@ object HiveExample {
 
     logger.info(s"============HIVE CLI IS STARTED ON PORT $port ==============")
 
-    statement.execute("CREATE TABLE IF NOT EXISTS " + "HIVE_CARBON_EXAMPLE " +
-                      " (ID int, NAME string,SALARY double)")
-    statement
-      .execute(
-        "ALTER TABLE HIVE_CARBON_EXAMPLE SET FILEFORMAT INPUTFORMAT \"org.apache.carbondata." +
-        "hive.MapredCarbonInputFormat\"OUTPUTFORMAT \"org.apache.carbondata.hive." +
-        "MapredCarbonOutputFormat\"SERDE \"org.apache.carbondata.hive." +
-        "CarbonHiveSerDe\" ")
+    statement.execute(
+      s"""
+         | CREATE TABLE IF NOT EXISTS HIVE_CARBON_EXAMPLE
+         | (ID int, NAME string,SALARY double)
+         | ROW FORMAT SERDE 'org.apache.carbondata.hive.CarbonHiveSerDe'
+         | WITH SERDEPROPERTIES ('mapreduce.input.carboninputformat.databaseName'='default',
+         | 'mapreduce.input.carboninputformat.tableName'='HIVE_CARBON_EXAMPLE')
+       """.stripMargin)
+
+    statement.execute(
+      s"""
+         | ALTER TABLE HIVE_CARBON_EXAMPLE
+         | SET FILEFORMAT
+         | INPUTFORMAT \"org.apache.carbondata.hive.MapredCarbonInputFormat\"
```

Comment:
This is fine for this PR, and after merging #3004 this ALTER TABLE will not be required

Reply:
Yeah, I really agree with you.But this pull request #3004 doesn't work in HiveExample.

Reply:
@jackylk Yeah, I really agree with you.But this pull request #3004 doesn't work in HiveExample.